contracts/Campaign.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT 
+pragma solidity ^0.8.28;
+
+contract HooslingCampaign {
+    
+    struct Campaign {
+        uint id;
+        address creator;
+        string title;
+        string description;
+        uint goal;
+        uint fundsRaised;
+        uint deadline;
+        bool withdrawn;
+    }
+
+    struct Contribution {
+        address donor;
+        uint amount;
```

Comment:
do we need receiver here? or project?

Reply:
Good observation. Currently, the Contribution struct only tracks the donor and amount. Adding a receiver or a projectId would provide clarity on where the funds are intended to go.

Reply:
we can add address or uints for projects Ids